scripts/parse_prophecy.py

```diff
@@ -58,15 +58,15 @@ def fastq_file_name_to_sample_id(self, filename: str) -> str:
         """
         return '_'.join(filename.split('_')[2:4])
 
-    async def get_read_filenames(self, sample_id: str, row: GroupedRow) -> List[str]:
+    async def get_read_filenames(self, sample_id: str, row: SingleRow) -> List[str]:
         """
         We don't have fastq urls in a manifest, so overriding this method to take
         urls from a bucket listing.
         """
         return [
             path
             for filename, path in self.filename_map.items()
-            if self.fastq_file_name_to_sample_id(filename) == sample_id
+            if self.fastq_file_name_to_sample_id(filename) == row['Sample/Name']
```

Comment:
This `row['Sample/Name']` should be defined as a constant somewhere - is it not the sample_id though?